frontend/packages/process-editor/src/components/Canvas/BPMNViewer/BPMNViewer.test.tsx

```diff
@@ -1,44 +1,19 @@
 import React from 'react';
 import { render, screen, act } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { BPMNViewer, BPMNViewerProps } from './BPMNViewer';
-import { textMock } from '../../../../../../testing/mocks/i18nMock';
-
-const mockAppLibVersion8: string = '8.0.1';
-const mockAppLibVersion7: string = '7.0.1';
-
-const defaultProps: BPMNViewerProps = {
-  appLibVersion: mockAppLibVersion8,
-};
+import { BPMNViewer } from './BPMNViewer';
 
 describe('Viewer', () => {
   afterEach(jest.clearAllMocks);
 
-  it('displays version alert when version is 7 or older', async () => {
+  it('should render the BPMN viewer', async () => {
     const user = userEvent.setup();
-    render(<BPMNViewer {...defaultProps} appLibVersion={mockAppLibVersion7} />);
+    render(<BPMNViewer />);
 
     // Fix to remove act error
     await act(() => user.tab());
 
-    const alertTitle = screen.getByRole('heading', {
-      name: textMock('process_editor.too_old_version_title'),
-      level: 1,
-    });
+    const alertTitle = screen.getByRole('link', { name: 'Powered by bpmn.io' });
```

Comment:
Nice, that we can render the `BPMNViewer` without JSDom issues, not sure if this works that great with `BPMNEditor`. 🙈

Reply:
I tested, and it does not 😢